.eslintrc.json

```
@@ -6,7 +6,9 @@
     "./header.eslintrc"
   ],
   "rules": {
-    "no-restricted-globals": ["error", "globalThis"]
+    "no-restricted-globals": ["error", "globalThis"],
+    "spaced-comment": ["error", "always", {"exceptions":["/"]}],
+    "capitalized-comments": ["error", "always", {"ignoreConsecutiveComments": true }]
```

I saw e.g. `// tslint:disable-next-line` in some tests and not sure if the capitalization will invalidate that. If so, we could add it to the [ignorePattern](https://eslint.org/docs/latest/rules/capitalized-comments#ignorepattern). But maybe those tslints can even be removed?
```suggestion
    "capitalized-comments": ["error", "always", {
      "ignorePattern": "tslint",
      "ignoreConsecutiveComments": true
    }]
```